dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -676,6 +683,10 @@ internal class CoreFeature(
     // endregion
 
     companion object {
+        internal const val SDK_INITIALIZED_IN_SECONDARY_PROCESS_WARNING_MESSAGE =
+            "Datadog SDK was initialized in a secondary process: although data will still be captured," +
+                "nothing will be uploaded from this process. Make sure to also initialize the SDK from the main" +
+                " process of your application."
```

Comment:
Question: does "the main process" refers to "main thread" in Android?

Reply:
In Android each process has its own JVM running code, each with several threads.